collector/cmd/dcos-metrics-collector/config_test.go

```diff
@@ -9,15 +9,15 @@ import (
 )
 
 func TestDefaultConfig(t *testing.T) {
-	testConfig := defaultConfig()
+	testConfig := newConfig()
 
 	testFileType := reflect.TypeOf(testConfig)
-	if testFileType.Name() != "ConfigFile" {
+	if testFileType.Name() != "CollectorConfig" {
```

Comment:
I'm not sure why I put this here, given the promises of strict typing and all :)